drivers/wifi/nrfwifi/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
```

Comment:
Can find no such mention of this commit in the linked upstream PR

Reply:
oops, it should be https://github.com/zephyrproject-rtos/zephyr/pull/79572 PR, will fix it thanks.